lib/index.js

```diff
@@ -205,7 +205,10 @@ function createPrefix(loaderContext, globalVarTree, globalVars, useEval) {
 module.exports = function loader(originalSource, inputSourceMap) {
     const self = this;
     const callback = this.async();
-    let source = originalSource;
+    let source = originalSource
+        //.replace(/\/\/.*/g, '') // remove line comments (somehow this bugs)
+        .replace(/\/\*[^]*?\*\//g, '/* stripped comment */'); // block comments
```

Comment:
Is `[^]*` equivalent to `.*`? More readable as the latter if so

Reply:
Hi
I just took people comments and made a PR. I was not even sure that you would answer. Now that I know I will take the time to understand and fix everything :)
Thank you for your feedback